soh/src/overlays/actors/ovl_En_Mag/z_en_mag.c

```diff
@@ -759,7 +746,7 @@ void EnMag_DrawInnerMq(Actor* thisx, PlayState* play, Gfx** gfxp) {
                           0);
         gDPSetPrimColor(gfx++, 0, 0, 0, 0, 0, textAlpha);
 
-        length = GetCharArraySize(noControllerMsg[lang]);
+        length = strlen(noControllerMsg[lang]);
```

Comment:
while i agree we should move away from using a custom method for this, it feels like an extra thing in this PR and makes it harder to review

this has been the case with some of your previous PRs as well. i appreciate the effort, but it helps a ton when reviewing to have PRs that stay focused on one thing. that way we can get changes we *definitely* want merged quickly, and not hold them up because they're coupled to changes that are less clear-cut

Reply:
Ok, I'll move it to another PR.